lib/events.js

```diff
@@ -344,6 +345,9 @@ EventEmitter.init = function(opts) {
       this._events === ObjectGetPrototypeOf(this)._events) {
     this._events = { __proto__: null };
     this._eventsCount = 0;
+    this[kShapeMode] = false;
```

Comment:
This has a comment above it asking to also update a definition in domain.js

Reply:
Nothing to change there as far as I can see.